pkgs/data/fonts/culmus/default.nix

```diff
@@ -3,17 +3,24 @@
 let
   version = "0.133";
 in fetchzip {
-  name = "culmus-${version}";
+  name = "culmus-${version}-1";
 
   url = "mirror://sourceforge/culmus/culmus/${version}/culmus-${version}.tar.gz";
 
   postFetch = ''
     tar -xzvf $downloadedFile --strip-components=1
-    mkdir -p $out/share/fonts/truetype
+    mkdir -p $out/share/fonts/{truetype,type1}
+    cp -v *.pfa $out/share/fonts/type1/
+    cp -v *.afm $out/share/fonts/type1/
+    cp -v fonts.scale-type1 $out/share/fonts/type1/fonts.scale
     cp -v *.ttf $out/share/fonts/truetype/
+    cp -v *.otf $out/share/fonts/truetype/
+    cp -v fonts.scale-ttf $out/share/fonts/truetype/fonts.scale
+    mkdir -p $out/etc/fonts/conf.d
+    cp -v culmus.conf $out/etc/fonts/conf.d/39-culmus.conf
   '';
 
-  sha256 = "1jxg2wf4kwasp5cia00nki2lrcdnhsyh4yy7d05l0a9bim5hq2lr";
+  sha256 = "1lvwv15lpk4cqarh2ncl83c43fmqxnzqaqzfm251zkx6svi1l0is";
```

Comment:
Something strange is going on here with the hash. It already changed during the last update: https://github.com/NixOS/nixpkgs/pull/50759

Reply:
Followed nix complaint about a bad hash

Reply:
I think I have an idea what is going here. Instead of using `mkDerivation` the whole thing just uses `fetchzip`. The hash is formed after the fetch is finished. So everytime something in the script `postFetch` is changed, the hash changes. The hash is not formed anymore for the downloaded source.
Would you mind rewriting it as a derivation? The whole thing should look something like this (schematically):
```
in stdenv.mkDerivation {
  name = ...

  src = fetchzip {
    url = ...
    sha256 = ...
  };

  installPhase = ''
   ...
  '';

  meta = ...
}
```
What is now in `postFetch` goes into `installPhase` of the derivation.